examples/subplots_axes_and_figures/secondary_axis.py

```diff
@@ -57,15 +57,19 @@ def rad2deg(x):
 ax.set_title('Random spectrum')
 
 
-def forward(x):
-    return 1 / x
+def safe_inverse(x):
+    x = np.array(x).astype(float)
```

Comment:
So I guess I changed the name because in the context of a tutorial that's trying to teach the user that you can pass any old function (as long as you also pass its inverse), it's *doubly* confusing that the function's inverse is itself in this case, but then that also the function is *called* "inverse". 

I just changed the name to `one_over(x)` explicitly, so that can be the name of "forward" and then the other one being called "inverse" is less confusing now IMHO without putting implementation details into the name of the function, is that better for you as well?

Reply:
If you told me that name is "too cute", I wouldn't disagree xD

Reply:
That's a reasonable compromise.

Reply:
Suggest if this was confusing that you change to a radians per second to period in seconds.  That way you get a factor that makes the forward and the inverse different.